src/content/PrivacyPolicy.md

```diff
@@ -0,0 +1,162 @@
+# Privacy Policy 
+
+Effective date: &lt;Date&gt;
+
+  
+
+Sage Bionetworks (“Sage Bionetworks, “we,” or “us”) is a 501(c)(3) nonprofit biomedical research organization in the United States.
+
+Sage Bionetworks is running the elevateMS research study sponsored by [Novartis Pharmaceuticals Corporation](https://www.pharma.us.novartis.com/). In the elevateMS research study, we are looking at how changes in MS symptoms affect daily life and wellness.  We use the elevateMS mobile application to measure these changes.  If you volunteer in this study we will ask you to use the elevateMS app. You can learn more about the study on the elevateMS website.  
```

Comment:
right now the statement says "elevateMS", should this get replaced with mPower 2.0?